README.md

```diff
@@ -42,8 +42,7 @@ $ cmake ..
 $ cmake --build .
 ```
 
-This will build Fizzy as a library and since there is no public API
-(the so called *embedder API* in WebAssembly) yet, this is not very useful.
+This will build Fizzy as a library. [C API] is provided for embedding Fizzy engine in applications, as well as [Rust binding].
```

Comment:
For C perhaps having a simple example would be useful, just to give an indication. Something which loads a wasm file and executes ta function called "main"?

Reply:
We also have the how to include fizzy in cmake. We have a test for that.

Reply:
@gumb0 perhaps this cmake example would be useful. See `test/integration/cmake_package/use_fizzy`.

Reply:
I'm not sure yet how to describe it other than mentioning `find_package(fizzy CONFIG REQUIRED)`. Can look into it later, or @chfast perhaps could help.

Reply:
Added a sentence anyway.